### ISSUE TITLE
table not found error on CREATE TABLE .. LIKE

### ISSUE THREAD

**Issue by flaneur2020** [datafuselabs/databend#7455]
**Summary**

```
use wubx;
create table t99( a int);
create table t100 like t99; -- failed
create table t100 like wubx.t99; -- success
select * from t99; -- success
drop table t99; -- success
```

**Comment by flaneur2020** [comment]
@BohuTANG says this issue is not reproduced in the localhost mysql.

**Comment by BohuTANG** [comment]
my local:
```
mysql> explain create table txx2 like txx1;
+---------------------------------------------------------------------------------------------------------------------------------------+
| explain                                                                                                                               |
+---------------------------------------------------------------------------------------------------------------------------------------+
| CreateTablePlanV2 { if_not_exists: false, tenant: "admin", catalog: "default", database: "default", table: "txx2", cluster_keys: [] } |
+---------------------------------------------------------------------------------------------------------------------------------------+
```

**Comment by sundy-li** [comment]
duplicated issue, fixed in #7497
